Resources/doc/resource_owners/zenmoney.md

```diff
@@ -0,0 +1,24 @@
+Step 2x: Setup Zenmoney
+=======================
+First you will have to register your application on Zenmoney. Check out the
```

Comment:
Please add a blank line before this one.

Reply:
That will not match all the rest files.
Shouldn't they all use same formatting?
Or do you want me to fix them all?